### PR TITLE
data, static/images/customers: add the NetEase case study

### PR DESCRIPTION
**Pull request by CaitinChen**

### CODE REVIEW

**Comment by CaitinChen** [comment]
@YiniXu9506 @queenypingcap PTAL

**Comment by CaitinChen** [comment]
@lilin90 PTAL

File: data/success_stories.json [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
     "name": "Internet",
     "children": [ 
+      {
+        "name": "NetEase Game",
+        "industry": "Gaming",
+        "websiteLink": "https://www.neteasegames.com/",
+        "link": "https://pingcap.com/success-stories/why-we-chose-a-distributed-sql-database-alongside-mysql-to-break-down-data-silos/",
+        "logo": "/images/customers/netease-games-logo.png",
+        "isShow": true,
+        "summary": "As one of the biggest players in China's online gaming market, NetEase Games has deployed TiDB in the production environment to service billing applications. With TiDB, NetEase Games is able to gain insights from data and create value for its users. It also plans to move more services to TiDB in the future."
```
Comment (lilin90) [inline data/success_stories.json]:
```suggestion
        "summary": "As one of the biggest players in China's online gaming market, NetEase Games has deployed TiDB in the production environment to serve billing applications. With TiDB, NetEase Games is able to gain insights from data and create value for its users. They also plan to move more services to TiDB in the future."
```

File: data/success_stories.json [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
     "name": "Internet",
     "children": [ 
+      {
+        "name": "NetEase Game",
+        "industry": "Gaming",
+        "websiteLink": "https://www.neteasegames.com/",
+        "link": "https://pingcap.com/success-stories/why-we-chose-a-distributed-sql-database-alongside-mysql-to-break-down-data-silos/",
+        "logo": "/images/customers/netease-games-logo.png",
+        "isShow": true,
+        "summary": "As one of the biggest players in China's online gaming market, NetEase Games has deployed TiDB in the production environment to service two applications. TiDB enables them to gain insights from data and create value for their users. They plan to move more services to TiDB in the future."
```
Comment (YiniXu9506) [inline data/success_stories.json]: If the summary here is what you want to display on the scale, please change it to 'scale'

**Pull request merged**